WWDC/SessionViewModel.swift

```diff
@@ -149,8 +149,9 @@ final class SessionViewModel {
     }()
 
     lazy var rxRelatedSessions: Observable<Results<RelatedResource>> = {
-        let predicateFormat = "type == %@ AND (ANY session.instances.sessionType == %d OR ANY session.instances.startTime >= %@)"
-        let relatedPredicate = NSPredicate(format: predicateFormat, RelatedResourceType.session.rawValue, SessionInstanceType.session.rawValue, today() as NSDate)
+        // Related must either have a video or occur in the future
+        let predicateFormat = "type == %@ AND (ANY session.assets.rawAssetType == %@ OR ANY session.instances.startTime >= %@)"
+        let relatedPredicate = NSPredicate(format: predicateFormat, RelatedResourceType.session.rawValue, SessionAssetType.sdVideo.rawValue, today() as NSDate)
```

Comment:
This makes me a little nervous in case the SD option goes away haha. 😅 But I'm not sure what a better way of determining this is. A `hasVideo` property which checks if there's a livestream, SD, or HD asset?

Reply:
Why not use `streamingVideo`? That also determines if the session has a video and is way less likely to go away in the foreseeable future.

Reply:
Also, I don't understand the "or occur in the future" part.

Reply:
Good shout.

“Occur in the future” is just me trying to understand whatever the existing logic was supposed to do. It’s trying to do “display anything that has a video, or any type of event that hasn’t yet occurred”, right?

Reply:
Okay, I've tried to clarify the comment as well as made the change